packages/coinstac-ui/app/render/components/consortium/consortium-results.js

```diff
@@ -1,33 +1,47 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import { Alert } from 'react-bootstrap';
+import { setExpandedResult } from '../../state/ducks/consortia-page';
 
 import ConsortiumResult from './consortium-result';
 
-export default class ConsortiumResults extends Component {
+class ConsortiumResults extends Component {
   constructor(props) {
     super(props);
 
-    // TODO: `state` holds collapsed/expanded bool for remote results. Move to redux.
-    this.state = {
-      expanded: [true],
-    };
+    this.goToResults = this.goToResults.bind(this);
   }
 
-  toggleCollapse(index) {
-    if (index in this.state.expanded) {
-      this.state.expanded[index] = !this.state.expanded[index];
+  componentWillMount() {
+    const { initialResultId, setExpandedResult } = this.props;
+
+    setExpandedResult(initialResultId);
+  }
+
+  goToResults() {
+    const { expandedResult } = this.props;
+
+    // Hacky - used to delay querying ids until results list items are rendered
```

Comment:
I agree: not ideal, but works for now. Could you change this comment to include the word `TODO:`? That way, when we search the codebase we can easily identify this.

Good example: https://github.com/MRN-Code/coinstac/blob/eb1c71bb394e091e811a1cfa3f665c559aa31d65/packages/coinstac-ui/app/render/components/settings.js#L24-L25